package/kernel/linux/modules/netdevices.mk

```diff
@@ -343,6 +427,23 @@ endef
 
 $(eval $(call KernelPackage,phy-marvell-10g))
 
+
+define KernelPackage/phy-micrel
+   SUBMENU:=$(NETWORK_DEVICES_MENU)
+   TITLE:=Micrel PHYs
+   KCONFIG:=CONFIG_MICREL_PHY
+   DEPENDS:=+kmod-libphy +kmod-ptp
```

Comment:
Do we always need the dependency to `kmod-ptp`? The target only set `CONFIG_PTP_1588_CLOCK_OPTIONAL` in its configuration, this dependency will add the full ptp framework. 

I do not know how to fix this. I think we had similar problems already in other places.

Reply:
maybe we can also ignore this for now and hope that this kernel package is only added to devices which are actually using the micrel phy chip.